src/utilities/CustomHooks.jsx

```diff
@@ -0,0 +1,283 @@
+import { useEffect } from "react";
```

Comment:
Create a hooks folder, and then have files in them. Eg., in this file you have useFetchingData(),  create a file in that name, have the functionality, and then call the hook from App.js.

Reply:
Instead I m doing this on the loading of the explore page or Home Page as of now they both are same .
The idea you are suggesting can be good but as of now I m gonna follow What I have been doing Since I have a deadline quite neat and lots of work to be done